packages/x-tree-view/src/internals/plugins/useTreeViewLabel/useTreeViewLabel.types.ts

```diff
@@ -76,5 +76,10 @@ export type UseTreeViewLabelSignature = TreeViewPluginSignature<{
   experimentalFeatures: 'labelEditing';
   dependencies: [UseTreeViewItemsSignature];
 }>;
-export interface UseTreeItem2LabelInputSlotPropsFromItemsReordering
```

Comment:
Typo in the name and the props were never added to the typing of `UseTreeItem2LabelInputSlotProps`